### PR TITLE
[11.0] [ADD] stock_account_change_qty_reason

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Depends on #524 
Allow to set accounts to  post the inventory changes on them apart of product or categories ones.

### CODE REVIEW

**Comment by HviorForgeFlow** [comment]
cc @jbeficent

File: oca_dependencies.txt [hunk 1]
```diff
@@ -4,3 +4,4 @@ purchase-workflow
 web
 server-ux
 account-analytic
+stock-logistics-warehouse https://github.com/Eficent/stock-logistics-warehouse.git 11.0-imp-stock_cange_qty_reason
```
Comment (LoisRForgeFlow) [inline oca_dependencies.txt]: Can you update this?

Comment (HviorForgeFlow) [reply oca_dependencies.txt]: Done :+1:

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

File: stock_account_change_qty_reason/views/stock_inventory_line_reason_view.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Eficent Business and IT Consulting Services S.L.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
```
Comment (rousseldenis) [inline stock_account_change_qty_reason/views/stock_inventory_line_reason_view.xml:3]: @hveficent Licensing are inconsistent inside module.

File: stock_account_change_qty_reason/views/stock_inventory_line_reason_view.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+<odoo>
+    <record id="view_account_preset_reason" model="ir.ui.view">
+        <field name="name">Preset Reason Accounts</field>
+        <field name="model">stock.inventory.line.reason</field>
+        <field name="inherit_id" ref="stock_change_qty_reason.view_preset_reason"/>
+        <field name="arch" type="xml">
+            <field name="description" position="after">
+                <field name="account_reason_input_id"/>
```
Comment (rousseldenis) [inline stock_account_change_qty_reason/views/stock_inventory_line_reason_view.xml]: @hveficent Maybe these fields should be visible for account people only ?

**Review by rousseldenis — CHANGES_REQUESTED** [review]
LGTM. Minor changes.

Dependency is merged

File: stock_account_change_qty_reason/views/stock_inventory_line_reason_view.xml (hunk 2 above)
Comment (HviorForgeFlow) [reply stock_account_change_qty_reason/views/stock_inventory_line_reason_view.xml:3]: All moved to AGPL

File: stock_account_change_qty_reason/views/stock_inventory_line_reason_view.xml (hunk 3 above)
Comment (HviorForgeFlow) [reply stock_account_change_qty_reason/views/stock_inventory_line_reason_view.xml]: now visible for account users at least

Comment (rousseldenis) [reply stock_account_change_qty_reason/views/stock_inventory_line_reason_view.xml]: How? Do not see changes

Comment (HviorForgeFlow) [reply stock_account_change_qty_reason/views/stock_inventory_line_reason_view.xml]: Now they are, missed them on previous push

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**